src/ProjectTemplates/Web.ProjectTemplates/content/Api-CSharp/Company.ApiApplication1.http

```diff
@@ -0,0 +1,11 @@
+@Company.ApiApplication1_HostAddress = http://localhost:5000
+
+Get {{Company.ApiApplication1_HostAddress}}/todos/
```

Comment:
All the examples in the VS Code REST Client https://marketplace.visualstudio.com/items?itemName=humao.rest-client  have the request method in all upper case. I think it would be better to have these in upper case.